src/main/resources/application.conf

```diff
@@ -1,7 +1,18 @@
-http {
+cromiam {
+  user_id_header = "X-User-Id"
+  allowed_users = [ "yossi" ]
```

Comment:
Yeah this is going to have to be changed for deployment, my goal was to make it a config change rather than a code change.